io.github.ec_.Quake3e.OpenArena.yaml

```diff
@@ -0,0 +1,77 @@
+app-id: io.github.ec_.Quake3e.OpenArena
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+# CHANGED: Let it look up the command via $PATH
+command: openarena-q3e
+finish-args:
+  # X11 + XShm access + Wayland
+  - --share=ipc
+  - --socket=wayland
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --socket=fallback-x11 
+ # CHANGED: Needs a persistent location for Quake 3 data files the user
+  # has permission to add things to and this is the only one in Quake3e's
+  # search path.
+  - --persist=.q3a
+  # CHANGED: Removed --filesystem=xdg-data because the Flathub submission
+  #          guides explicitly tell you NOT to try to preserve access to the
+  #          un-sandboxed versions of paths like ~/.local/share. Users can
+  #          copy/move their stuff to ~/.var/app/org.fxt.openarena/data
+  #          manually if they want to.
+modules:
+  # CHANGED: Module name isn't displayed to users and convention says lowercase
+  - name: openarena
+    buildsystem: simple
+    sources:
+      # CHANGED: Pull direct from the upstream source
+      - type: git
+       # only-arches: x86_64
```

Comment:
```suggestion
```

Reply:
I want to leave it there for use with test builds, so they take me less time

Reply:
it is invalid anyway. flatpak-builder throw a warning about it.

Reply:
I already commented it, if it complains about commits, flatpak builder is wrong